RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -35,7 +35,9 @@ def _updateOutput( era, outputPSets, commands):
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+from Configuration.Eras.Modifier_bParking_cff import bParking
 _outputs = [RecoLocalMuonFEVT, RecoLocalMuonRECO, RecoLocalMuonAOD]
+_updateOutput(bParking, RecoLocalMuonAOD, ['keep *_dt1DRecHits_*_*', 'keep *_csc2DRecHits_*_*'])
```

Comment:
```suggestion
_updateOutput(bParking, [RecoLocalMuonAOD], ['keep *_dt1DRecHits_*_*', 'keep *_csc2DRecHits_*_*'])
```
I guess this is going to fix the crashes.
Please double-check in a local test (parsing a config is enough)